src/elm/Page/Register.elm

```diff
@@ -172,108 +151,197 @@ view guest model =
-                    , span [] [ text_ "register.account_created.account" ]
-                    , span [] [ text_ "register.account_created.created" ]
+
+        passphraseTextId =
+            "passphraseText"
+
+        passphraseInputId =
+            -- Passphrase text is duplicated in `input:text` to be able to copy via Browser API
+            "passphraseWords"
+
+        viewTitleForStep : Int -> Html msg
+        viewTitleForStep s =
+            let
+                step =
+                    String.fromInt s
+            in
+            p
+                [ class "py-4 mb-4 text-body border-b border-dotted text-grey border-grey-500" ]
+                [ text (tr "register.form.step" [ ( "stepNum", step ) ])
+                , text " / "
+                , strong
+                    [ class <|
+                        if s == 1 then
+                            "text-black"
+
+                        else
+                            "text-white"
                     ]
-                , div [ class "register__dog" ] []
+                    [ text_ ("register.form.step" ++ step ++ "_title") ]
                 ]
-            , div [ class "register__welcome" ]
-                [ text (tr "register.account_created.welcome_message" [ ( "username", name ) ]) ]
-            , div [ class "register__keys" ]
-                [ p [ class "key__title" ] [ text_ "register.account_created.twelve_words" ]
-                , p [ class "key", id "12__words" ] [ text (words model) ]
-                , p [ class "key__title" ] [ text_ "register.account_created.private_key" ]
-                , p [ class "key", id "p__key" ] [ text (privateKey model) ]
+    in
+    if model.accountGenerated then
+        div
+            [ class "flex-grow bg-purple-500 flex md:block"
+            ]
+            [ div
+                [ class "sf-wrapper"
+                , class "px-4 md:max-w-sm md:mx-auto md:pt-20 md:px-0 text-white text-body"
                 ]
-            , div [ class "register__instructions" ]
-                [ p [] [ text_ "register.account_created.instructions" ] ]
-            , button
-                [ onClick DownloadPdf
-                , class "btn btn__register"
+                [ div [ class "sf-content" ]
+                    [ viewTitleForStep 2
+                    , p
+                        [ class "text-xl mb-3" ]
+                        [ text_ "register.account_created.greet"
+                        , text " "
+                        , strong [] [ text name ]
+                        , text ", "
+                        , text_ "register.account_created.last_step"
+                        ]
+                    , p [ class "mb-3" ]
+                        [ text_ "register.account_created.instructions"
+                        ]
+                    , div
+                        [ class "w-1/4 m-auto relative"
+                        , style "left" "1rem"
```

Comment:
maybe we can achieve the same result if we only have a `left-0` class with a `pl-4` or `ml-4`?

I would prefer to avoid `style` most of the time

Reply:
Found out that we already have `left-1` class :)